databases/redis/Portfile

```diff
@@ -24,37 +25,8 @@ set redis_domain    redis.io
 homepage            https://${redis_domain}
 master_sites        https://download.${redis_domain}/releases/
 
-checksums           rmd160  86968a20c18ee7a9383edcec1237d750f01ef0a7 \
-                    sha256  4ddebbf09061cbb589011786febdb34f29767dd7f89dbe712d2b68e808af6a1f \
-                    size    3533864
-
-patchfiles          patch-redis.conf.diff \
-                    patch-hiredis.diff \
-                    patch-gh-12585.diff
-
-# https://github.com/redis/redis/pull/13534
-patchfiles-append   0001-config.h-fix-Availability-macros-so-they-actually-wo.patch \
-                    0002-debug.c-do-not-use-arm-code-on-ppc.patch \
-                    0003-debug.c-fix-PowerPC-case-in-uc_mcontext.patch
-
-# ld: warning: directory '/usr/local/opt/openssl/lib' following -L not found
-patchfiles-append   0004-Remove-brewisms.patch
-
-post-patch {
-    reinplace "s|@PREFIX@|${prefix}|g" \
-        ${worksrcpath}/redis.conf \
-        ${worksrcpath}/deps/hiredis/Makefile
-}
-
-# see ${worksrcpath}/deps/jemalloc/Makefile
-compiler.cxx_standard   2014
-configure.cxxflags-append   -std=c++14
-
-# https://github.com/redis/redis/issues/13535
-if {[string match *gcc* ${configure.compiler}]} {
-    configure.ldflags-append \
-                    -latomic
-}
+# Global settings common to both subports
+revision            0
```

Comment:
the `revision` field is specific to each port, so should be set in both subport separately